### PR TITLE
feat: dining car stuff

### PR DESCRIPTION
**Pull request by midgleyc**
Don't have the autoassembly yet.

### CODE REVIEW

File: src/data/modifiers.txt [hunk 1]
```diff
@@ -2714,6 +2714,7 @@ Item	astral ring	Mysticality Percent: +25, Spell Damage: +20, MP Regen Min: 4, M
 Item	atlas of local maps	Combat Rate: -5, Lasts Until Rollover
 # attorney's badge: OBJECTION!
 Item	attorney's badge	Single Equip
+Item	automatic wine thief	Single Equip
```
Comment (Veracity0) [inline src/data/modifiers.txt:2717]: Doesn't the description tell you what this does so that you can put a comment here? (Same for the other equipment)

**Review by Veracity0 — CHANGES_REQUESTED** [review]

**Comment by Veracity0** [comment]
Is auto assembly real? Last night after rollover I was in trade getting my last trainbot skills and somebody wrote "Irrat just made the furst autoassembler" (or something.) I PMed Irrat and he said "he frll for a fake screenshot".

**Comment by midgleyc** [comment]
There are a few people on the forum thread claiming to have got one (or many).

Apparently it's not marked public yet.

**Review by Veracity0 — APPROVED** [review]

**Comment by Veracity0** [comment]
> There are a few people on the forum thread claiming to have got one (or many).
> 
> 
> 
> Apparently it's not marked public yet.

Wild speculation:

It opens shop.php where you can make things out of Trainbot parts.

**Pull request merged**